packages/quantic/cypress/e2e/default-1/tab-bar/tab-bar-selectors.ts

```diff
@@ -21,7 +21,8 @@ export const TabBarSelectors: TabBarSelector = {
   allTabs: () => TabBarSelectors.get().find(tabComponent),
   activeTab: () =>
     TabBarSelectors.get().find('button.slds-tabs_default__item.slds-is-active'),
-  moreButton: () => TabBarSelectors.get().find('.tab-bar_more-button'),
+  moreButton: () =>
```

Comment:
So that one didn't work with the `getByRole` I assume?

Reply:
nope, thats why I settled for the `getByTestId`